src/BloomBrowserUI/pageChooser/js/page-chooser.ts

```diff
@@ -111,8 +135,8 @@ class PageChooser {
             var dataBookArray = $( "div[data-book='bookTitle']", pageData );
             var collectionTitle = $( dataBookArray.first() ).text();
             // Add title and container to dialog
-            var collectionToAdd = $( collectionHTML).clone();
-            $( collectionToAdd ).find( ".collectionCaption" ).text(collectionTitle);
+            var collectionToAdd = $(collectionHTML).clone();
+            this.setLocalizedText($(collectionToAdd).find(".collectionCaption"), 'Template.', collectionTitle);
```

Comment:
Except for AddPageDialog.Description, all the localization IDs (and various translations of them) I used here already existed. So I was just reusing, not designing, at least by intent. I didn't research where else they are used, but if we're thinking of changing them we will need to.